src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.CodeGeneration.targets

```diff
@@ -117,7 +124,11 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="_ResolveGeneratedRazorCompileInputs" BeforeTargets="ResolveRazorCompileInputs">
+  <PropertyGroup>
+    <ResolveRazorCompileInputsDependsOn>$(ResolveRazorCompileInputsDependsOn);_ResolveGeneratedRazorCompileInputs</ResolveRazorCompileInputsDependsOn>
```

Comment:
Just my personal preference to use DependsOn properties over `Before` \ `After` Targets. It also makes the order of it's execution more deterministic.